src/WebexSDKAdapter.js

```diff
@@ -29,5 +32,6 @@ export default class WebexSDKAdapter {
   async disconnect() {
     await this.sdk.internal.mercury.disconnect();
     await this.sdk.internal.device.unregister();
+    this.meetingAdapter.disconnect();
```

Comment:
```suggestion
    await this.meetingAdapter.disconnect();
```

Reply:
Is this defined anywhere? If you look at [CI test report](https://app.circleci.com/jobs/github/webex/sdk-component-adapter/360/parallel-runs/0/steps/0-104), integration tests are failing because `meeting.disconnect` is not defined

Reply:
added that